README.md

```diff
@@ -17,11 +17,11 @@ Rx support with [react-native-contacts-rx](https://github.com/JeanLebrument/reac
 
 
 ## API
-`getAll` (callback) - returns *all* contacts as an array of objects  
```

Comment:
looks like the whitespace removed here removed the newlines? These should each be rendered on their own line.